Automattic/newspack-plugin
# Add more fonts to design options

### All Submissions:






### Changes proposed in this Pull Request:

Adds 4 new Google Fonts to the Design Wizard:
* Bodoni Moda (serif)
* DM Sans (sans serif)
* Jost (sans serif)
* Newsreader (serif)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Go to the Design Wizard and change the fonts for Headings and Body
3. Publish the changes and check the front-end

### Other information:
